frontend/src/bundles/workouts/components/create-workout-form/constants/constants.ts

```diff
@@ -0,0 +1,46 @@
+import { ActivityType } from '~/bundles/common/enums/enums.js';
+import {
+    capitalizeFirstLetter,
+    configureDateString,
+    configureTimeString,
+} from '~/bundles/common/helpers/helpers.js';
+import { type CreateWorkoutPayload } from '~/bundles/workouts/types/types.js';
+
+const DEFAULT_CREATE_WORKOUT_PAYLOAD: CreateWorkoutPayload = {
+    activityType: ActivityType.CYCLING,
+    workoutDate: configureDateString(new Date().toISOString()),
+    workoutStartedAt: configureTimeString(new Date().toISOString()),
+    workoutEndedAt: '',
+    speed: '',
+    distance: '',
+    steps: '',
+    heartRate: '',
+    kilocalories: '',
+    provider: null,
+};
+
+const WORKOUT_ACTIVITY_OPTIONS = [
```

Comment:
I think we already have similar selector with ActivityType somewhere, maybe we could reuse options?